examples/article-access-laterpay.amp.html

```diff
@@ -233,15 +236,8 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
           </div>
         </div>
 
-        <section id="meter-notif" amp-access="views > 0" amp-access-hide class="meter-section">
-          <template amp-access-template type="amp-mustache">
-            You are viewing article {{views}} of {{maxViews}}!
-          </template>
-          <a on="tap:meter-notif.hide">[Close]</a>
-        </section>
-
         <section amp-access="NOT error AND NOT access" amp-access-hide class="login-section">
-          <a on="tap:amp-access.login">Login to read more!</a>
+          <amp-access-laterpay-list></amp-access-laterpay-list>
```

Comment:
1. Given that people can potentially customize the look and feel of it, some might prefer displaying it as a dialog while others might prefer displaying it as just a section below an article. But in any case, I don't have any strong preference for one name or the other.
2. Right now, no, and we'd like to give the users some measure of control over where this shows up, rather than just having a dialog pop up over all of their content. What we do in our existing client side integration is to have them specify a selector for an element over which our dialog will be placed. So we also don't have a "default" action there, it's always something that requires that little bit of configuration. AFAIK so far this hasn't been a major issue specially because the publishers we've worked with so far seem to like to have that measure of control. 
3. Either would be fine, also don't have a special preference on this. Just assumed this would be the best way for doing this in AMP.